### PR TITLE
advent-2018: Monte Carlo and Pi

### PR DESCRIPTION
**Pull request by sbinet**

### CODE REVIEW

File: content/advent-2018/montecarlo.md [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+gauss: mean=    10.105225624460644
+gauss: std-dev= 5.048629091912316 +/- 0.05048629091912316
+```
+
+![plot-gauss](/postimages/advent-2018/montecarlo/gauss.png)
+
+Ok, so now we know how to generate random numbers that follow some distribution.
+How do we evaluate π with that?
+
+## Computing^WApproximating π using a Monte Carlo technique
```
[the remark this answers is not in the document]
Comment (sbinet) [reply content/advent-2018/montecarlo.md]: what do you mean?
I get this:
approx

[the remark this answers is not in the document]
Comment (btracey) [reply content/advent-2018/montecarlo.md]: Then let me rephrase, I don't know what Computing^WApproximating is supposed to mean

[the remark this answers is not in the document]
Comment (sbinet) [reply content/advent-2018/montecarlo.md]: ah :)
`^W` erases a word in POSIX shells.
perhaps needlessly geeky...

File: content/advent-2018/montecarlo.md [hunk 2]
```diff
@@ -0,0 +1,829 @@
...
+
+![plot-gauss](/postimages/advent-2018/montecarlo/mc-pi.png)
+
+## Conclusions
+
+I hope this quick foray into a technique that is at the heart of many physics simulations was fun.
+The Monte Carlo technique isn't always the fastest technique to perform simulations (this obviously depends on the "shape" of the function we want to model) but it is deceptively simple, and can be visually "fun" -- for some definition of _fun_.
+
+Note that in this little example, we had fun creating a PNG image, encoding it to JSON+`base64` and sending it over a WebSocket.
+This was just to exercize a bunch of packages with Go.
```
Comment (Pisush) [inline content/advent-2018/montecarlo.md]: s/exercize/exercise (i think?)

File: content/advent-2018/montecarlo.md [hunk 3]
```diff
@@ -0,0 +1,829 @@
...
+[Gonum/plot](https://godoc.org/gonum.org/v1/plot/vg) has support for a few backends in addition to PNG: JPEG, TIFF, EPS, PDF, SVG, TeX.
+
+The complete code for `mc-pi-plot.go` is here:
```
Comment (Pisush) [inline content/advent-2018/montecarlo.md]: maybe it's more readable as a link to your repo?

File: content/advent-2018/montecarlo.md (hunk 2 above)
Comment (sbinet) [reply content/advent-2018/montecarlo.md]: It's the American English spelling.
(Which has been drilled down in my skull after many article reviews :P)

File: content/advent-2018/montecarlo.md (hunk 3 above)
Comment (sbinet) [reply content/advent-2018/montecarlo.md]: Good point.
I'll link to a gist.

Comment (sbinet) [reply content/advent-2018/montecarlo.md]: done.
